readme.txt

```diff
@@ -2,8 +2,8 @@
 Contributors: wpengine, markkelnar, stevenkword, stephenlin, ryanshoover, taylor4484
 Tags: wpe, wpengine, geoip, localization, geolocation
 Requires at least: 3.0.1
-Tested up to: 4.8
-Stable tag: 1.2.1
+Tested up to: 4.8.1
```

Comment:
So, this is a little WP trick.  If we specify the minor version, the plugin directory will specifically use that version.  When 4.8.2 drops, we'd have to update again.

However, if you only use the major version, 4.8, the directory will show it compatible w/ 4.8.2 automagically.